### PR TITLE
Automate building of packages in OBS 

### PR DESCRIPTION
**Pull request by aazam-gh**
This PR is part of the Feilong Packaging Process to automate building of packages (zthin and zvmsdk) for the 3 distros (Ubuntu, RHEL, SLES) on OBS.

In order for this automation to work, we must use a webhook in order for OBS to receive events from the github action to trigger the rebuild of the packages with the latest source files. 
In order to use the webhook, we require a Personal Access Token (PAT) which will be used by OBS to generate the url for the webhook.

The working of the action is as follows: 
- When build-zvmsdk receives the dispatched event from this (feilong) github action, it first runs a few script files to generate new tarballs from the latest version of the feilong master branch.
- After creating all the source files, it downloads them as artifacts and pushes them to the master branch of build-zvmsdk.
- Once build-zvmsdk has a push, it triggers the webhook with OBS which causes each package hosted on OBS to download the required tarballs and/or .dsc files followed by triggering a rebuild for each package.

This [PR](https://github.com/openmainframeproject/feilong/pull/839) handles the development on `feilong` side.

Please create a PAT on github for this repository.
The webhook will then be created with the `Push` event.

### CODE REVIEW

File: zthin-build.tar.gz [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
+<html><head>
+<title>404 Not Found</title>
+</head><body>
+<h1>Not Found</h1>
+<p>The requested URL was not found on this server.</p>
+<hr>
+<address>Apache/2.4.52 (Ubuntu) Server at download.kde.org Port 443</address>
+</body></html>
```
Comment (mfriesenegger) [inline zthin-build.tar.gz]: Should this file be removed from the PR?

**Comment by mfriesenegger** [comment]
@aazam-gh @Bischoff Is best practice to squash all of the commits for a PR?

File: zthin-build.tar.gz (hunk 1 above)
Comment (aazam-gh) [reply zthin-build.tar.gz]: Yes.

Comment (Bischoff) [reply zthin-build.tar.gz]: Yes. Unless you really meant to have different commits addressing distinct topics.

**Comment by aazam-gh** [comment]
Closed in favour of https://github.com/mfcloud/build-zvmsdk/pull/10 due to rebase conflicts

**Pull request closed without merging**